os/dbuild.sh

```diff
@@ -154,6 +157,14 @@ function SELECT_OPTION()
 	fi
 }
 
+function BUILD_TEST()
+{
+	echo "Build Test START!! Docker Image Version : ${DOCKER_VERSION}"
+	pushd ${OSDIR} > /dev/null
+	docker run --rm ${DOCKER_OPT} -v ${TOPDIR}:/root/tizenrt -w /root/tizenrt/os --privileged tizenrt/tizenrt:${DOCKER_VERSION} bash -c "./tools/build_test.sh"
```

Comment:
If we make this separately, when we need to modify docker command, we should modify two different lines. How about making a new function for this and call the same function at here and build function for maintainability?

Reply:
will modify them with another PR.